docs/tutorials/aws/organizations.md

```diff
@@ -51,6 +59,79 @@ The additional fields in CSV header output are as follows:
 - ACCOUNT\_DETAILS\_ORG
 - ACCOUNT\_DETAILS\_TAGS
 
+## Deploying Prowler IAM Roles Across AWS Organizations
+
+When onboarding multiple AWS accounts into Prowler Cloud, it’s important to deploy the Prowler Scan IAM Role in each account. The most efficient way to do this across an AWS Organization is by leveraging AWS CloudFormation StackSets, which allows you to roll out infrastructure—like IAM roles—to all accounts centrally from your Management or Delegated Admin account.
+
+If you're using Infrastructure as Code (IaC), we recommend using Terraform to manage this deployment systematically.
+
+### Recommended Approach
+
+- **Use StackSets** from the **Management Account** (or a Delegated Admin/Security Account).
+- **Use Terraform** to orchestrate the deployment.
+- **Use the official CloudFormation template** provided by Prowler.
+- Target specific Organizational Units (OUs) or the entire Organization.
```

Comment:
As there is no other approach I'm not sure about keeping this section.

Reply:
They could do it manually, but we recommend this approach.